plugin-base/src/main/java/io/everytrade/server/plugin/impl/everytrade/parser/exchange/binance/v4/BinanceSupportedOperations.java

```diff
@@ -0,0 +1,75 @@
+        unsupportedAccountTypes.add(BinanceConstantsV4.ACCOUNT_TYPE_POOL.code);
+        unsupportedAccountTypes.add(BinanceConstantsV4.ACCOUNT_TYPE_USDT_FUTURES.code);
+
+        //unsupported operations
+        unsupportedOperationTypes.add(BinanceConstantsV4.OPERATION_TYPE_MARGIN_LOAN.code);
+        unsupportedOperationTypes.add(BinanceConstantsV4.OPERATION_TYPE_MARGIN_REPAYMENT.code);
+        unsupportedOperationTypes.add(BinanceConstantsV4.OPERATION_TYPE_BNB_DEDUCT_FEE.code);
+        unsupportedOperationTypes.add(BinanceConstantsV4.OPERATION_TYPE_ISOLATED_MARGIN_LOAN.code);
+        unsupportedOperationTypes.add(BinanceConstantsV4.OPERATION_TYPE_ISOLATED_MARGIN_REPAYMENT.code);
+        unsupportedOperationTypes.add(BinanceConstantsV4.OPERATION_TYPE_TRANSFER_IN.code);
+        unsupportedOperationTypes.add(BinanceConstantsV4.OPERATION_TYPE_TRANSFER_OUT.code);
+        unsupportedOperationTypes.add(BinanceConstantsV4.OPERATION_TYPE_P2P_TRADING.code);
+        unsupportedOperationTypes.add(BinanceConstantsV4.OPERATION_TYPE_POOL_DISTRIBUTION.code);
+        unsupportedOperationTypes.add(BinanceConstantsV4.OPERATION_TYPE_BNB_DEDUCTS_FEE.code);
+        unsupportedOperationTypes.add(BinanceConstantsV4.OPERATION_TYPE_COMMISSION_FEE_SHARED_WITH_YOU.code);
+        unsupportedOperationTypes.add(BinanceConstantsV4.OPERATION_TYPE_COMMISSION_HISTORY.code);
+        unsupportedOperationTypes.add(BinanceConstantsV4.OPERATION_TYPE_DISTRIBUTION.code);
+        unsupportedOperationTypes.add(BinanceConstantsV4.OPERATION_TYPE_LARGE_OTC_TRADING.code);
+        unsupportedOperationTypes.add(BinanceConstantsV4.OPERATION_TYPE_LAUNCHPOOL_INTEREST.code);
+        unsupportedOperationTypes.add(BinanceConstantsV4.OPERATION_TYPE_LIQUID_SWAP_ADD.code);
+        unsupportedOperationTypes.add(BinanceConstantsV4.OPERATION_TYPE_POS_SAVINGS_INTEREST.code);
+        unsupportedOperationTypes.add(BinanceConstantsV4.OPERATION_TYPE_POS_SAVINGS_PURCHASE.code);
+        unsupportedOperationTypes.add(BinanceConstantsV4.OPERATION_TYPE_POS_SAVINGS_REDEMPTION.code);
+        unsupportedOperationTypes.add(BinanceConstantsV4.OPERATION_TYPE_SAVINGS_INTEREST.code);
+        unsupportedOperationTypes.add(BinanceConstantsV4.OPERATION_TYPE_SAVINGS_PRINCIPAL_REDEMPTION.code);
+        unsupportedOperationTypes.add(BinanceConstantsV4.OPERATION_TYPE_SAVINGS_PURCHASE.code);
+        unsupportedOperationTypes.add(BinanceConstantsV4.OPERATION_TYPE_SMALL_ASSET_EXCHANGE_BNB.code);
+        unsupportedOperationTypes.add(BinanceConstantsV4.OPERATION_TYPE_SUPER_BNB_MINING.code);
+        unsupportedOperationTypes.add(BinanceConstantsV4.OPERATION_TYPE_FUNDING_FEE.code);
+        unsupportedOperationTypes.add(BinanceConstantsV4.OPERATION_TYPE_INSURANCE_FUND_COMPENSATION.code);
+        unsupportedOperationTypes.add(BinanceConstantsV4.OPERATION_TYPE_REALIZE_PROFIT_AND_LOSS.code);
+        unsupportedOperationTypes.add(BinanceConstantsV4.OPERATION_TYPE_REFEREE_REBATES.code);
+    }
+}
```

Comment:
here github complains that file doesnt end with empty line. its common practice to have new line at end of each file in git.